index.html

```diff
@@ -404,8 +404,7 @@ <h2>
           readonly attribute WakeLockType type;
           readonly attribute boolean active;
           attribute EventHandler onactivechange;
-          Promise&lt;void&gt; request();
-          void abort();
+          Promise&lt;void&gt; request(optional WakeLockRequestOptions options);
```

Comment:
this should be passed in the constructor IMO.

Reply:
Please file an issue for that... it also depends on the later cancellation discussion, so let's look at that after

Reply:
Need to think about it... can a WakeLock be reused after .request()?

Reply:
yes it can

Reply:
hmm.. IMHO, you should only be able to use a WakeLock once. Otherwise it leads to more insanity around aborting.

Reply:
This is really another discussion we should have in another issue

Reply:
sure, np.

Reply:
https://github.com/w3c/wake-lock/issues/195